contracts/WildcardSteward_v2.sol

```diff
@@ -102,6 +108,18 @@ contract WildcardSteward_v2 is Initializable {
         _;
     }
 
+    modifier priceGreaterThanZero(uint256 _newPrice) {
+        require(_newPrice > 0, "Price is zero");
+        _;
+    }
+    modifier validWildcardsPercentage(uint256 wildcardsPercentage) {
+        require(
+            wildcardsPercentage >= 500 && wildcardsPercentage <= 10000,
```

Comment:
Make max 100% - artist commission

Reply:
Have fixed this modifier. I have also made the artist commission configurable for the future. Right now it defaults to 1%, but for each token (i.e. for each specific art piece), we can change this amount. Maybe we will have artist profiles etc and those with higher quality works etc might get higher commissions etc. Think its useful having it configurable so we can strike different deals with different entities.